packages/fuels/tests/predicates/predicate_address/src/main.sw

```diff
@@ -1,10 +1,7 @@
 predicate;
 
-use std::inputs::input_predicate_data;
+fn main(input: Address) -> bool {
```

Comment:
Do you mind adding one last test where `main` has a single argument that is a copy type (such as `u64`). Sorry if you've added that already and I missed it.

Reply:
We do not have a test like that (there is one with two unsigned ints). I will add it :)